test.js

```diff
@@ -0,0 +1,24 @@
+const codeImport = require('./');
+const remark = require('remark');
+
+const input = `
+\`\`\`js file=./__fixtures__/say-hi.js
+\`\`\`
+`;
+
+test('Basic file import', () => {
+  expect(
+    remark()
+      .use(codeImport, {})
+      .processSync({
+        contents: input,
+        path: __dirname + '/say-hi.js',
```

Comment:
I think this is wrong, this should be the path of the markdown file. Maybe just `test.md`? And prefer to use standard `path` library to create path string.

```suggestion
        path: path.join(process.cwd(), 'test.md'),
```